issuetracker/src/main/java/org/openjdk/skara/issuetracker/jira/JiraIssue.java

```diff
@@ -181,6 +181,21 @@ public State state() {
         }
     }
 
+    /**
+     * A Jira issue is considered fixed if it's either resolved or closed and
+     * the resolution is "Fixed".
+     */
+    @Override
+    public boolean isFixed() {
+        if (isResolved() || isClosed()) {
```

Comment:
This test might not be needed -- the resolution will be empty or null for unresolved issues. I don't see any harm in leaving it if you prefer to have the extra check.

Reply:
I agree that JBS should guarantee this to be true. I went back and forth on this myself, but in the end I think it's clearer like this.